### DISCUSSION TITLE
Conflicting macro definitions in directory macros/adapters

### DISCUSSION THREAD

**Discussion by JonathanCauchi** [dbt-athena/dbt-athena#752] [Q&A] [answered]
Hi all,

I'm working on configuring dbt with AWS Athena. It seems that the profiles.yml configuration is successful as all checks pass after running dbt debug. However, when performing a dbt run. I get this issue:

```
Compilation Error
  dbt found two macros named "athena__get_columns_in_relation" in the project
  "dbt_athena".
   To fix this error, rename or remove one of the following macros:
      - macros\adapters\columns.sql
      - macros\adapters.sql
```

When I remove one conflicting macro, it mentions another macro in a different file. I suspect it has something to do with the dependency versions. The following are the dependency versions I'm working with:

```
dbt-adapters               1.7.0
dbt-athena                 0.0.5
dbt-athena-community       1.8.4
dbt-common                 1.11.0
dbt-core                   1.8.1
pyathena                   2.25.2
```

Could anyone assist?

**Comment by nicor88** [comment, marked as answer]
Looks like that you have an old version of dbt-athena installed (0.0.5). Please disinstall that, and keep only dbt-athena-community 1.8.4.
The issue is most likely related to the old athena package taking priority over dbt-athena-community.
